src/main/java/org/kaazing/netx/ws/internal/WsURLConnectionImpl.java

```diff
@@ -435,13 +435,27 @@ public void doFail(int code, String exceptionMessage) throws IOException {
         throw new IOException(exceptionMessage);
     }
 
-    public WebSocketContext getContext(WebSocketExtensionSpi sentinel, boolean reverse) {
+    public DefaultWebSocketContext getIncomingContext() {
```

Comment:
Is the `WebSocketContext` interface sufficient to return here instead of the implementation?